cmd/minishift/cmd/start.go

```diff
@@ -353,6 +353,7 @@ func initClusterUpFlags() {
 	clusterUpFlagSet.Int(serverLogLevel, 0, "Log level for the OpenShift server.")
 	clusterUpFlagSet.StringSliceVarP(&openShiftEnv, openshiftEnv, "e", []string{}, "Specify key-value pairs of environment variables to set on the OpenShift container.")
 	clusterUpFlagSet.Bool(metrics, false, "Install metrics (experimental)")
+	clusterUpFlagSet.Bool(logging, false, "Install logging (experimental)")
```

Comment:
@praveenkumar thanks, that's what I was looking for. I was not sure how to access Kibana. Trying ....